src/constructorio.js

```diff
@@ -30,6 +31,31 @@ const computePackageVersion = () => {
   return `${versionPrefix}${versionModifiers.join('-')}${versionModifiers.length ? '-' : ''}${packageVersion}`;
 };
 
+const setSessionId = (sessionId, options) => {
```

Comment:
I could be missing something, but is there a reason why we aren't using the identity module to handle setting the session id?

I am slightly concerned that we need to reproduce the logic that currently exists in our identity module. I feel like most of the logic that we're introducing here should already be handled by the identity module itself and if it doesn't have support for what we need, we should be making adjustments there instead of the JS client.

We have access to the identity module from the JS client and I believe we have access to it's properties (i.e. `cookie_name_session_id`, `local_name_session_id`, etc) and the functions to set the cookie (i.e. `set_cookie`).

We should schedule a quick discussion to chat through the current implementation and see if it's possible to utilize the identity module to handle this.